src/main/java/Parser.java

```diff
@@ -0,0 +1,93 @@
+public class Parser {
+
+    /** TaskList class that stores and deals with the tasks **/
+    private TaskList taskList;
+
+    /**
+     *Class constructor
+     *
+     */
+    public Parser(TaskList taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Parse the inputs and carry out the respective commands
+     *
+     * @param inputs the array of command and taskDescription obtain from user
+     */
+    public void parseCommand(String[] inputs) {
```

Comment:
Perhaps can break up this method into shorter methods?

Reply:
> Perhaps can break up this method into shorter methods?

I agree with @xxzz-tt. Perhaps you could abstract away lines 24 - 28 to be in the respective task commands?